synapseclient/table.py

```diff
@@ -787,10 +839,10 @@ class Dataset(ViewBase):
     Folders can easily be added recursively to a dataset, that is, all files
     within the folder (including sub-folders) will be added.  Note that using
     the following methods will add files with the latest version number ONLY.
-    If another version number is desired, use :py:meth:`synapseclient.table.add_item`
-    or :py:meth:`synapseclient.table.add_items`.
+    If another version number is desired, use [add_item][synapseclient.table.add_item]
+    or [add_items][synapseclient.table.add_items].
 
-    Example::
+    Example:
```

Comment:
Double check how these are rendered on the mkdocs page to verify they show up like we want them to.